examples/Getting Started/Export Image.ipynb

```diff
@@ -0,0 +1,223 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x222d40bbf60>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# preview image being exported\n",
```

Comment:
The preview cell seems out of place. Shouldn't you preview the image before you export it?

Reply:
well, not if your driving...